addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1974,6 +1981,47 @@ QUnit.test('focusing a chat window of a chat should make new message separator d
     );
 });
 
+QUnit.test('chat window: click on correspondent name in chat header, redirects to user form', async function (assert) {
+    assert.expect(4);
+
+    this.data['res.partner'].records.push({ id: 7, name: "Demo" });
+    const chat = {
+        channel_type: "chat",
+        is_minimized: true,
+        members: [this.data.currentPartnerId, 7],
+    };
+    this.data['mail.channel'].records.push(chat);
+
+    const PartnerFormPromise = testUtils.makeTestPromise();
```

Comment:
prefer "deferred" name instead of "promise" when the resolve is leaked and called manually later

Also don't start variable name with a capital letter.